packages/sdk-btc/source/wallet-discovery.service.test.ts

```diff
@@ -0,0 +1,249 @@
+import "jest-extended";
+
+import nock from "nock";
+
+import { identity } from "../test/fixtures/identity";
+import { createService } from "../test/mocking";
+import { WalletDiscoveryService } from "./wallet-discovery.service";
+import { BindingType } from "./constants";
+import { AddressFactory } from "./address.factory";
+
+beforeAll(async () => {
+	nock.disableNetConnect();
+});
+
+describe("testnet", () => {
+	const subject = createService(WalletDiscoveryService, "btc.testnet", (container) => {
+		container.singleton(BindingType.AddressFactory, AddressFactory);
+	});
+
+	it("should generate an output from a mnemonic", async () => {
+		const result = await subject.fromMnemonic(identity.mnemonic);
+
+		expect(result).toEqual([
+			{
+				address: "mjmLqx9vd9d3ANn7eX3tc7cdJhWqt3V722",
```

Comment:
@faustbrian I didn't know how to generate these ones outside of our code to validate (like I did with the rest).

Reply:
They're pretty old and no longer supported to be generated so will have to check how to create those somewhere.